pkg/controller/servicebindingrequest/retriever_test.go

```diff
@@ -16,6 +16,8 @@ func TestRetriever(t *testing.T) {
 	ns := "testing"
 	backingServiceNs := "backing-servicec-ns"
 	crName := "db-testing"
+	crId1 := "testingCrId1"
+	crId2 := "testingCrId2"
```

Comment:
There should be tests for these scenarios as well:

[1] What happens when id is an empty string `""`
[2] What happens when id is not unique like:
```
crId1 = "testingCrId1"
crId2 = "testingCrId1"
```

Reply:
fixed